build/templates/errors.py.mako

```diff
@@ -95,6 +95,15 @@ class SelfTestError(Error):
         super(SelfTestError, self).__init__('Self-test failed with code {0}: {1}'.format(code, msg))
 
 
+% endif
+% if 'DriverTooNewError' in extra_errors_used:
+class DriverTooNewError(Error):
+    '''An error due to the driver runtime being too new compared to the Python module'''
+
+    def __init__(self, prefix_message):
+        super(DriverTooNewError, self).__init__(f'{prefix_message}This can occur if the ${module_name} Python module is too old. Upgrade the ${module_name} Python module.')
```

Comment:
In general we structure error elaborations as follows:

1. what happened
2. what the customer can do

```suggestion
        super(DriverTooNewError, self).__init__(f'{prefix_message}An unexpected value was returned from the ${driver_name} runtime. This can occur if the ${driver_name} runtime is newer than the ${module_name} Python module. Upgrade the ${module_name} Python module.')
```